src/unit4/lesson1/README.md

```diff
@@ -0,0 +1,88 @@
+# Unit 4, Lesson 1 - Setting up your first cluster
+
+Hello and welcome to Unit 4. You know a lot of advanced features of RavenDB. Now it is time to start learning about how to use clusters.
+
+To be honest, it is not a difficult task.  In fact, you do not need to be an expert to manage a RavenDB cluster (We did our best to make it mostly self-managed).
+
+## What is the point of setting up a cluster?
+
+A RavenDB cluster is a set of machines that have been joined together. Each machine in the cluster is a node.
+
+When you create a database, it can live on a single node (one machine in the cluster), some number of the nodes or even all the nodes. Each node will then hold a complete copy of the database and will be able to serve all queries, operations and writes (RavenDB cluster implements ![multi-master replication](https://en.wikipedia.org/wiki/Multi-master_replication).
+
+The primary reason for this duplicating data is to allow high availability. If a  ode goes down, the cluster will still have a copies of the data, and the cluster can shuffle things around so clients can talk to another node without really noticing that anything happened.
+
+The cluster can distribute the work, handle the failures and adopt recovery procedures automatically.
+
+## So, let’s set up a cluster?
+
+In fact, assuming that you are already started the RavenDB server on your computer,  you are already using a Cluster (a cluster with a single node, but it is still a cluster).
+
+![You already have a cluster](media/already_cluster.png)
+
+The thing you need to do is to add more nodes to your cluster (I will help you)
+
+## So, could we simply add nodes to a cluster?
+
+Yes. We Can! But, first we need to review our license: RavenDB Community edition only allows you to set up the cluster using 3 cores (total). So, before setting a cluster, we will need to reduce the number of cores in use of the current node. You can skip this depending on your license.
+
```

Comment:
Let's mention here that you can use a developer license that have much higher number of cores.
Might be more appropriate for this.